### DISCUSSION TITLE
How to connect to olad

### DISCUSSION THREAD

**Discussion by brandhout** [mikacousin/olc#23] [Q&A]
Hi!

I'm unable to use olc. OLA is running, I can control the lights with the web and CLI interface but the flatpak of OLC is just.... empty. Maybe I'm doing something wrong, but the app doesn't seem connected to OLA. Typing 1 Ch 100 @ doesn't do anything.

How can I at least verify that OLC is connected to OLA ? Do I need to select the right universe somewhere? 
![screenshot](https://github.com/mikacousin/olc/assets/23652365/87c51f99-b28a-4f13-9473-1edbd8d33b85)

**Comment by mikacousin** [comment]
Hi,

My first idea is that by default, olc uses the sacn python module.

To use ola, run olc from the command line with :
```bash
$ olc --backend ola
```
You only have to do it once, and olc will save it for later.

Then, as you can see in the define.py file, olc uses universes 1, 2, 3 and 4 by default.
If you want to use universe 0, you have to edit this file.

Please keep me informed.

**Reply by brandhout** [reply]
Thanks! That seems to work. Unfortunately ola seems to be poorly packaged/ buggy on my distro, GNU Guix, so I've sent some patches which are dependent on a libusb update, so that might take some time.  

I've also packaged python-sacn and have a work in progress  but working olc package definition for Guix, so that might appear someday.  

I will keep you informed.

**Comment by mikacousin** [comment]
Great, I'll let you mark the discussion as resolved if you think that's the case.
And thanks for your work on the packages.

Just for the record, the flathub version of olc embeds ola directly. And olad is launched automatically if needed (with the ola backend, of course).
